setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# W391 blank line at end of file
+# E302 expected 2 blank lines, found 1
+ignore = W391,E302
```

Comment:
I have participated in several attempts in bringing these sorts of "style unifications" into code bases. They have all ended in massive bikeshedding disasters and lost time and energy for everyone involved. The only way to make this work is to use Clang-format (which does not do Python) and have computers do it a) automatically b) reliably. No existing tool can do that for Python code.